### PR TITLE
spelling fixes (non-trivial, changing messages)

### PR DESCRIPTION
**Pull request by ka7**
##### SUMMARY

patch contains some spelling fixes as found by a bot ( http://www.misfix.org, https://github.com/ka7/misspell_fixer ). Any upcoming License changes (e.g. GPL2 to GPL3+) are hereby granted.

Fhe fixes touching spelling-fixes in output, so _might break_ some stuff if somebody relays one that ??

##### ISSUE TYPE

 - Feature Pull Request
 - New Module Pull Request
 - Bugfix Pull Request
 - Docs Pull Request

##### ANSIBLE VERSION

```

```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: lib/ansible/modules/source_control/github_release.py [hunk 1]
```diff
@@ -91,8 +91,7 @@ def main():
     )
 
     if not HAS_GITHUB_API:
-        module.fail_json(msg='Missing required github3 module (check docs or '
```
Comment (Akasurde) [inline lib/ansible/modules/source_control/github_release.py]: This change reverts #23641

**Review by Akasurde — CHANGES_REQUESTED** [review]
Except github_release change, I am OK with other changes.

**Comment by ka7** [comment]
sorry for the exclusion of that, error by me while rebase.. :-/
fixed that now.

File: lib/ansible/module_utils/aos.py [hunk 2]
```diff
@@ -158,7 +158,7 @@ def do_load_resource(module, collection, name):
     try:
         item = find_collection_item(collection, name, '')
     except:
-        module.fail_json(msg="Ans error occured while running 'find_collection_item'")
+        module.fail_json(msg="Ans error occurred while running 'find_collection_item'")
```
Comment (gundalow) [inline lib/ansible/module_utils/aos.py:161]: `An error occurred`

**Review by gundalow — CHANGES_REQUESTED** [review]

**Comment by gundalow** [comment]
Rebase needed

**Comment by calfonso** [comment]
@ka7 still interested in making these changes? If so, please update the PR and if not, you can close it. Thanks!

**Comment by Akasurde** [comment]
@ka7 are you still working on this ? I can take over this PR.

**Comment by ka7** [comment]
Hi,

sorry, got stuck a bit, as git-rebase trys to make me crazy.

If you want to take over - feel free.

.klemens

**Comment by Akasurde** [comment]
@ka7 OK. Thanks.

**Comment by gundalow** [comment]
Closing as this has been fixed in another PR

**Pull request closed without merging**